cmd/configure.ac

```diff
@@ -257,5 +257,22 @@ fi
 fi
 AM_CONDITIONAL([USE_INTERNAL_BPF_HEADERS], [test "x$host_bpf_headers_usable" = "xno"])
 
+AC_DEFINE([MISSING_FIELD_INITIALIZERS_USABLE], [1], [-Wmissing-field-initializers are usable])
```

Comment:
I would use `AC_SUBST` instead.

Reply:
Sorry nevermind, I meant `AM_CONDITIONAL`. And I see it is done later. Maybe we can just remove this line.